test/ngResource/resourceSpec.js

```diff
@@ -97,6 +97,43 @@ describe('basic usage', function() {
     $httpBackend.flush();
   });
 
+  it('should include a request body when calling custom delete with hasBody is true', function() {
```

Comment:
_custom delete_ --> _custom method_

Can you also test with another method (in the same test), just to avoid giving the impression this is just about `DELETE`.